tests/lib/find.test.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+var expect = require('expect.js'),
+    _ = require('underscore'),
+    find = require('../../src/lib/find');
+
+
+describe('find', function() {
+   it('returns undefined when given a non array-like parameter to search through', function() {
+      expect(find(undefined, undefined)).to.be(undefined);
+      expect(find(null, undefined)).to.be(undefined);
+      expect(find(false, undefined)).to.be(undefined);
```

Comment:
These tests are not validly testing what you think they're testing. As noted above, if you 'give a non array-like parameter to search through' there will be an error because there is no check in the code for the array other than array.length. This test is giving you a false sense of security because the undefined function is what's actually causing you to enter the `if` branch in your code.